src/pages/Admin/AdminVenuePreview.tsx

```diff
@@ -69,13 +67,13 @@ export const AdminVenuePreview: React.FC<AdminVenuePreview> = ({
         );
       case VenueTemplate.partymap:
       case VenueTemplate.themecamp:
-        const campVenue = venue as WithId<CampVenue>;
+        const partyMapVenue = venue as WithId<PartyMapVenue>;
```

Comment:
This type should move to `props` instead of `as` assignment

Reply:
![Screen Shot 2021-01-27 at 7 01 22 pm](https://user-images.githubusercontent.com/753891/105961003-1401e800-60d2-11eb-924f-6b87da641049.png)

Looks like it was done this way to work around TypeScript. There would be ways we could 'fix this properly', but since this is existing code that i've just updated the types on, I think I will call it OOS for this PR.

I expect that once we can delete all of the old code in https://github.com/sparkletown/sparkle/pull/833, it will be simpler to see how we can properly fix/unify/simplify our types.

Reply:
It's also possible that we might be able to refactor `PartyMapContainer` to be able to take `WithId<Venue>` directly.. which would solve this here. Will check if that breaks anything then comment back here.

Reply:
Looks like the changes are going to be annoying to make, and it's not a big enough issue to be worth blocking this PR over (as it's more of a typing issue than a 'real' issue, as we already check that it's one of the correct template types in the `case` statement before this code runs.

So going to call this OOS for this PR.